opensrp-anc/src/main/java/org/smartregister/anc/library/model/RegisterModel.java

```diff
@@ -60,4 +61,9 @@ public JSONObject getFormAsJson(String formName, String entityId, String current
     public String getInitials() {
         return Utils.getUserInitials();
     }
+
+    @VisibleForTesting
+    protected com.vijay.jsonwizard.utils.FormUtils getFormUtils() {
+        return new com.vijay.jsonwizard.utils.FormUtils();
+    }
```

Comment:
Can't we access this method directly? `com.vijay.jsonwizard.utils.FormUtils();`

Reply:
Updated